R/repl.R

```diff
@@ -5,11 +5,22 @@
 #' run within the Python 'main' module, and will remain accessible after the
 #' REPL is detached.
 #' 
-#' @param banner Boolean; print a startup banner when launching the REPL?
+#' @param quiet Boolean; print a startup banner when launching the REPL? If
+#'   `FALSE`, the banner will be suppressed.
+#' @param modules A set of Python modules (packages) to be imported before
+#'   the REPL is launched. Defaults to the value of the `reticulate.repl.modules`
+#'   option.
 #' 
 #' @export
-python_repl <- function(banner = TRUE) {
- 
+python_repl <- function(
```

Comment:
Now that I see `python_repl()`, I think we are better of going back to `py_repl()` for consistency with the other `py_` functions. I think if we could get away with something simple like `repl()` that would be better, but I think that would be over-reaching. Let me know if you agree.

Reply:
I just worry about in-theory masking of other functions called `repl()` in other packages, but maybe that's fine and we could / should just encourage people to call `reticulate::repl()` directly if that ever did come up.

Reply:
I agree about repl() being too generic. I also think we are going to want people to do library(reticulate) followed by py_repl() so that they can get access to py$ from R.

Reply:
I wonder if there is a Python function somewhere in their standard libraries that would give us the printing behavior we are looking for.

Reply:
I wonder if you should be using `py_eval()` (which returns the result of evaluating the expression) rather than `py_run_string()`

Reply:
`py_eval()` unfortunately doesn't work for certain kinds of Python statements; e.g.

```
> py_eval("x = 1")
Error in py_eval_impl(code, convert) : 
  SyntaxError: invalid syntax (reticulate_eval, line 1)
```